### ISSUE TITLE
transaction.created_time is always empty

### ISSUE THREAD

**Issue by kauffj** [lbryio/chainquery#33]
```
mysql> SELECT DISTINCT(DATE(created_time)) FROM transaction;
+----------------------+
| (DATE(created_time)) |
+----------------------+
| 1970-01-01           |
+----------------------+
1 row in set (3.89 sec)
```

**Comment by tiger5226** [comment]
field https://github.com/btcsuite/btcd/blob/master/rpcserverhelp.go#L198
being set https://github.com/lbryio/chainquery/blob/master/daemon/processing/transaction.go#L138

All results in DB have 1 or 2. Why?

Merging issue #30

**Comment by tiger5226** [comment]
We were setting nano seconds instead of seconds. This has been corrected going forward. However, we will need an upgrade script to set this value correctly for all transaction.

**Comment by tiger5226** [comment]
resolved with 92aa385d551173c8d9b7f8eb53dd2059d2ed6bfc

Once we finalize the schema based on https://github.com/lbryio/chainquery/issues/35, a reindex will be performed.
